code/datums/diseases/flu.dm

```diff
@@ -4,52 +4,52 @@
 	spread_text = "Airborne"
 	cure_text = "Spaceacillin"
 	cures = list(/datum/reagent/medicine/spaceacillin)
-	cure_chance = 10
+	cure_chance = 5
 	agent = "H13N1 flu virion"
 	viable_mobtypes = list(/mob/living/carbon/human)
 	permeability_mod = 0.75
 	desc = "If left untreated the subject will feel quite unwell."
 	severity = DISEASE_SEVERITY_MINOR
 
 
-/datum/disease/flu/stage_act()
+/datum/disease/flu/stage_act(delta_time, times_fired)
 	. = ..()
 	if(!.)
 		return
 
 	switch(stage)
 		if(2)
-			if(affected_mob.body_position == LYING_DOWN && prob(20))
+			if(affected_mob.body_position == LYING_DOWN && DT_PROB(10, delta_time))
 				to_chat(affected_mob, "<span class='notice'>You feel better.</span>")
 				stage--
 				return
-			if(prob(1))
+			if(DT_PROB(0.5, delta_time))
 				affected_mob.emote("sneeze")
-			if(prob(1))
+			if(DT_PROB(0.5, delta_time))
 				affected_mob.emote("cough")
-			if(prob(1))
+			if(DT_PROB(0.5, delta_time))
 				to_chat(affected_mob, "<span class='danger'>Your muscles ache.</span>")
 				if(prob(20))
 					affected_mob.take_bodypart_damage(1, updating_health = FALSE)
-			if(prob(1))
+			if(DT_PROB(0.5, delta_time))
 				to_chat(affected_mob, "<span class='danger'>Your stomach hurts.</span>")
 				if(prob(20))
 					affected_mob.adjustToxLoss(1, FALSE)
 
 		if(3)
-			if(affected_mob.body_position == LYING_DOWN && prob(15))
+			if(affected_mob.body_position == LYING_DOWN && DT_PROB(7.5, delta_time))
 				to_chat(affected_mob, "<span class='notice'>You feel better.</span>")
 				stage--
 				return
-			if(prob(1))
+			if(DT_PROB(0.5, delta_time))
 				affected_mob.emote("sneeze")
-			if(prob(1))
+			if(DT_PROB(0.5, delta_time))
 				affected_mob.emote("cough")
-			if(prob(1))
+			if(DT_PROB(0.5, delta_time))
 				to_chat(affected_mob, "<span class='danger'>Your muscles ache.</span>")
 				if(prob(20))
```

Comment:
Missed one? Maybe?

Reply:
No, the outer prob already fixes the time dependency, so using DT_PROB here would cause wrong behaviour.

Reply:
The outer one handles it. Applying it twice would halve it.